home.html

```diff
@@ -0,0 +1,187 @@
+           <a href="about.html"> <li class="navigationItem">
+                About
+            </li></a>
+           <a href="home.html#program"> <li class="navigationItem">
+                Program
+            </li></a>
+            <a href="home.html#join"> <li class="navigationItem">
+                Join
+            </li></a>
+           <a href="about.html#sponsor"> <li class="navigationItem">
+                Sponsor
+            </li>
+        </a>
+        </a>
+           <a href="#news"> <li class="navigationItem">
+                News
+            </li>
+        </a>
+        </ul>
+    </div>
+<header class="header flex">
+    <h1 class="mainTitle">
+        "Mellon!"
+    </h1>
+    <h2 class="secondTitle">
+        ARE YOU READY  A JOURNEY TO  MIDDLE-EARTH?
+    </h2>
+</header>
+<main>
+    <div class="index flex">
+        <p class="description">
+            As a Tolkien reader or  a Lord of the Rings watcher, we invite you a fantastic journey in Istanbul with different people all over the world.   
+        </p>
+        <p class="date">
+            2024.03.01 (WED) - 05 (FRI)</p>
+        <p class="location">
+@Taksim Square of İstanbul in Turkey
+        </p>
+    </div>
+    <div class="program flex" id="program">
+        <h2 class="mainProgram">
+            Main Program
+        </h2>
+        <hr class="mainHr"/>
+        <ul class="schedule flex">
+            <li class="scheduleItems flex">
+                <img src="lecture-logo.png" class="scheduleItemImage">
+                <h3 class="scheduleTitle"> 
+                    Opening
+                 </h3> 
+                 <p class="scheduleIndex">
+                    As speaker grandson of J.R.R Tolkien Simon Tolkien
+                 </p>
+        </li>
+        <li class="scheduleItems flex">
+            <img src="exhibition-logo.png" class="scheduleItemImage">
+            <h3 class="scheduleTitle"> 
+                Video Shows
+            </h3>
+            <p class="scheduleIndex">
+                The highlights of series
+            </p>
+        </li>
+        <li class="scheduleItems flex">
+            <img src="workshop-logo.png" class="scheduleItemImage">
+            <h3 class="scheduleTitle">
+                
+                Workshops
+            </h2>
+            <p class="scheduleIndex">
+                Role playing games, Creating Middle-Earth musics
+            </p>
+        </li>
+        <li class="scheduleItems flex">
+            <img src="network-logo.png" class="scheduleItemImage">
+            <h3 class="scheduleTitle"> 
+                Consers
+            </h3>
+            <p class="scheduleIndex">
+                Sting,Michael Jackson-Luciano Pavarotti, Nil Karaibrahimgil, Amy Winehouse
+            </p>
+        </li>
+        </ul>
+        <button class="joinButton" id="join">
+            Get Hurry to Join Us!
+        </button>
+<a href="#program"> <p class="wholeProgram">SEE THE WHOLE PROGRAM</p></a>
```

Comment:
- [ ] @otmanTR great work implementing the changes requested, however the mobile view still needs some margin between the header and image elements as , I have indicated in the image below to match the design given

![image](https://user-images.githubusercontent.com/61408860/212408404-950b4d67-9e21-471f-8dae-9da02fd2d2cd.png)

- [ ] @otmanTR as the previous reviewer noted your font color doesn't match the design given kindly add the recommended font in your project 
![image](https://user-images.githubusercontent.com/61408860/212409022-267aef0b-3feb-4e12-9433-1f3f28354513.png)